16_kata.py

```diff
@@ -0,0 +1,11 @@
+# https://www.codewars.com/kata/5552101f47fc5178b1000050/train/python
+
+def dig_pow(n, p):
+
+    list_variables = []
```

Comment:
This variable could have a better name to be fair. You store digits here that you sum in the end, but the name indicates you store variables. 

You could, for example, name it `total` and assign it `0` as initial value, and further in the code (line 7), instead of appending to the list, you could just add to `total`. Thanks to this we have a better naming and you don't need to use modules like `append` or `sum` later :)